.typos.toml

```diff
@@ -0,0 +1,51 @@
+[files]
+extend-exclude = [
+    ".git/",
+    # "bin/",
```

Comment:
@svetasmirnova bin contains a lot of typos fixed in lib.
Could it be that some were not updated?

Reply:
I need to check. But all modules should not be touched in bin. However, some tool's code exists in the bin version only (it starts from "package TOOLNAME;" line), so having line ending checks in bin would be nice.

Reply:
> having line ending checks in bin would be nice.

@svetasmirnova I am starting with EditorConfig.
Line ends are fixed in #664
There are thousand files with trailing whitespaces.

```
eclint -exclude="util/NaturalDocs/**" -exclude="bin/**" -summary | wc -l
1154
```
Would you welcome a PR with those removed?
The single command would be
```
eclint -exclude="util/NaturalDocs/**" -exclude="bin/**" -fix
```

Reply:
Yes, please, but check if the tests are passed with Percona Server 5.7 and 8.0. @alina-derkach-oaza - do we use trailing whitespaces in docs for something?

Reply:
@svetasmirnova, no, we usually delete trailing whitespaces in docs.